src/harness/fake_sas.py

```diff
@@ -206,16 +206,16 @@ def GetFullActivityDump(self, ssl_cert=None, ssl_key=None):
     response = json.loads(json.dumps({'files':[
              {'url': "https://raw.githubusercontent.com/Wireless-Innovation-Forum/\
              Spectrum-Access-System/master/schema/empty_activity_dump_file.json",
-              'checksum': "da39a3ee5e6b4b0d3255bfef95601890afd80709",'size':19, 'version': "v1.0",'recordType': "cbsd" },
+              'checksum': "da39a3ee5e6b4b0d3255bfef95601890afd80709",'size':19, 'version': "v1.2",'recordType': "cbsd" },
```

Comment:
Why not use self.version, as you do on line 381 below?

Reply:
done